src/main/java/jnr/constants/platform/darwin/ProtocolFamily.java

```diff
@@ -40,12 +40,57 @@
 PF_PPP(34L),
 // PF_ATM not defined
 // PF_NETGRAPH not defined
-PF_MAX(38L);
```

Comment:
38L vs 40L?

Reply:
Not sure if it changed or was manually added with an incorrect value. It does appear to be 40 on 10.13.

```
[] ~/projects/jnr-constants $ grep -r PF_MAX /usr/include
/usr/include/net/bpf.h:#define BPF_MAXINSNS 512
/usr/include/net/bpf.h:#define BPF_MAXBUFSIZE 0x80000
/usr/include/sys/socket.h:#define	PF_MAX		AF_MAX

[] ~/projects/jnr-constants $ grep -r AF_MAX /usr/include
/usr/include/sys/socket.h:#define	AF_MAX		40
/usr/include/sys/socket.h:#define	PF_MAX		AF_MAX
/usr/include/sys/socket.h:#define	NET_MAXID	AF_MAX
```